XenModel/FriendlyNames.resx

```diff
@@ -1845,7 +1845,7 @@
     <value>[Citrix] Virtual Apps and Desktops [Citrix] Cloud</value>
   </data>
   <data name="Label-host.edition-legacy-basic" xml:space="preserve">
-    <value>[Legacy Citrix XenServer] Basic Edition</value>
+    <value>[Citrix XenServer] Basic Edition</value>

```

Comment:
In retrospect, I believe this should have been `[Legacy XenServer product]` instead of `[Citrix XenServer]`. Will fix it now.